processor/spanmetricsprocessor/processor_test.go

```diff
@@ -323,6 +361,12 @@ func TestProcessorConsumeTraces(t *testing.T) {
 			verifier:               verifyConsumeMetricsInputDelta,
 			traces:                 []pdata.Traces{buildSampleTrace(), buildSampleTrace()},
 		},
+		{
+			name:                   "Test maximum span time will not cause out of bounds index error",
+			aggregationTemporality: delta,
+			verifier:               verifyMetricsNoOp,
```

Comment:
probably better to use an inline, anonymous function here

Reply:
fixed